dev_setup.sh

```diff
@@ -33,7 +33,12 @@ if [ $(id -u) -eq 0 ]; then
 fi
 
 TOP=$(cd $(dirname $0) && pwd -L)
-VIRTUALENV_ROOT=${VIRTUALENV_ROOT:-"${HOME}/.virtualenvs/mycroft"}
+
+if [ -z "$WORKON_HOME" ]; then
+    VIRTUALENV_ROOT=${VIRTUALENV_ROOT:-"${HOME}/.virtualenvs/mycroft"}
+else
+    VIRTUALENV_ROOT="$WORKON_HOME"
```

Comment:
It appears to me we should concatenate WORKON_HOME + /mycroft

`VIRTUALENV_ROOT="$WORKON_HOME/mycroft"`